src/app/events/Ready.ts

```diff
@@ -14,5 +16,6 @@ export default class implements DiscordEvent {
 		await connect(constants.DB_NAME_DEGEN);
 		await connect(constants.DB_NAME_BOUNTY_BOARD);
 		await GuestPassService(client);
```

Comment:
I just realized that this events should only be activated for Bankless DAO can you do

```
if (client.guilds.cache.some((guild) => guild.id == discordServerIds.banklessDAO || guild.id == discordServerIds.discordBotGarage)) {
			await connect(constants.DB_NAME_BOUNTY_BOARD);
			await GuestPassService(client);
			await restoreScoapEmbedAndVoteRecord();
		}
```

Reply:
done